src/main/webapp/js/userMap.js

```diff
@@ -1,18 +1,17 @@
-document.addEventListener('DOMContentLoaded', function(event) {
+function handleData(err, userData) {
     // based on example from https://github.com/markmarkoh/datamaps/blob/master/src/examples/highmaps_world.html
     // Country code: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
-
+    if (err) {
+        console.log('error in getting data');
```

Comment:
are there any standard styling that I can follow?
i.e. `class` and `<div>` or `<p>`, etc